### PR TITLE
improve API usage

### PR DESCRIPTION
**Pull request by rande**

### CODE REVIEW

File: public_keys.go [hunk 1]
```diff
@@ -29,6 +30,17 @@ func (g *Gitlab) UserKeys() ([]*PublicKey, error) {
 	return keys, err
 }
 
+func (g *Gitlab) ListKeys(id string) ([]*PublicKey, error) {
```
Comment (plouc) [inline public_keys.go]: You should update the test

File: users.go [hunk 2]
```diff
@@ -2,10 +2,11 @@ package gogitlab
 
 import (
 	"encoding/json"
+	"strconv"
 )
 
 const (
-	users_url        = "/users"     // Get users list
+	users_url        = "/users?page=:page&per_page=:per_page"     // Get users list
```
Comment (plouc) [inline users.go]: can you please align descriptions

**Comment by rande** [comment]
@plouc done

File: public_keys.go [hunk 3]
```diff
@@ -9,6 +9,7 @@ const (
 	// ID
 	user_keys        = "/user/keys"     // Get current user keys
 	user_key         = "/user/keys/:id" // Get user key by id
+	list_keys        = "/users/:uid/keys"
```
Comment (plouc) [inline public_keys.go]: Can you please add a description

**Pull request merged**